test/test_datapipe.py

```diff
@@ -589,19 +598,40 @@ def test_serializable(self):
         dp_skip_comparison = {dp.iter.FileLister, dp.iter.FileOpener, dp.iter.StreamReader, dp.iter.Shuffler}
         # These DataPipes produce multiple DataPipes as outputs and those should be compared
         dp_compare_children = {dp.iter.Demultiplexer, dp.iter.Forker}
+
         for dpipe, dp_args, dp_kwargs in picklable_datapipes:
             datapipe = dpipe(input_dp, *dp_args, **dp_kwargs)  # type: ignore[call-arg]
-            serialized_dp = pickle.dumps(datapipe)
-            deserialized_dp = pickle.loads(serialized_dp)
+            serialized_dp = pickle.dumps(datapipe)  # Making sure dpipe is picklable
+            _ = pickle.loads(serialized_dp)  # Making sure dpipe is loadable
```

Comment:
You can remove these two lines.

Reply:
I have refactored the code to make it clear what I'm doing